scss/_layouts_application.scss

```diff
@@ -89,19 +116,22 @@ $application-layout--side-nav-width-expanded: 15rem !default;
       width: $application-layout--side-nav-width-collapsed;
 
       &.is-collapsed {
+        overflow: hidden;
         transform: translateX(0);
         width: $application-layout--side-nav-width-collapsed;
       }
     }
 
     .l-navigation:hover {
       box-shadow: $panel-drop-shadow;
+      overflow-y: scroll;
```

Comment:
@sowasred2012 Would 'auto' work? I think we don't want scrollbar if it's not needed.